ui/components/MesheryApplications.js

```diff
@@ -395,19 +395,24 @@ function MesheryApplications({
 
   const handleAppDownload = (id, source_type, name) => {
     updateProgress({ showProgress : true })
-    dataFetch(
-        `/api/application/download/${id}/${source_type}`,
-        {
-          credentials : "include",
-          method : "GET",
-        },
-        () => {
-          fileDownloader(id, name, source_type);
-          console.log("ApplicationFile API", `/api/application/download/${id}/${source_type}`);
-          updateProgress({ showProgress : false });
-        },
-        handleError(ACTION_TYPES.DOWNLOAD_APP)
-    );
+    try {
```

Comment:
There should be a change in designs as well in UI.

Reply:
added for designs as well.